cgo/lapacke/generate_lapacke.go

```diff
@@ -0,0 +1,554 @@
+// Copyright ©2016 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build ignore
+
+// generate_lapacke creates a lapacke.go file from the provided C header file
+// with optionally added documentation from the documentation package.
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"go/format"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"text/template"
+
+	"github.com/cznic/cc"
+
+	"github.com/gonum/internal/binding"
+)
+
+const (
+	header        = "lapacke.h"
+	documentation = "../native"
```

Comment:
It would be nice to have documentation generation in the cgo package. I'm not sure if native's documentation couldn't be occasionally misleading here, for example about when a function panics, etc.